Ex3/GarageLogic/ValueOutOfRangeException.cs

```diff
@@ -13,6 +13,16 @@ public ValueOutOfRangeException(float i_MinValue, float i_MaxValue)
             r_MaxValue = i_MaxValue;
         }
 
+        public override string Message
+        {
+            get
+            {
+                return string.Format("Value out of range." + Environment.NewLine + 
+                    "Min value: {0}." + Environment.NewLine + 
+                    "Max value: {1}" + Environment.NewLine,
+                    this.MinValue, this.MaxValue);
+            }
+        }
```

Comment:
Should not be done this way - https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions

Reply:
also should have a blank line here